File: superstitioMain/java/superstitio/powers/masoOnly/LostBodyPart_Leg.java
package superstitio.powers.masoOnly;

import superstitio.DataManager;

import static superstitio.cards.general.FuckJob_Card.BodyPart;

public class LostBodyPart_Leg extends LostBodyPart {
    public static final String POWER_ID = DataManager.MakeTextID(LostBodyPart_Leg.class);

    public LostBodyPart_Leg() {
        super(POWER_ID);
    }

    @Override
    public BodyPart[] banedBodyPart() {
        return new BodyPart[]{BodyPart.leg};
    }
}

File: superstitioMain/java/superstitio/powers/masoOnly/LostBodyPart_Leg.kt
package superstitio.powers.masoOnly

import superstitio.DataManager
import superstitio.cards.general.FuckJob_Card.BodyPart

class LostBodyPart_Leg : LostBodyPart(POWER_ID)
{
    override fun banedBodyPart(): Array<BodyPart?>
    {
        return arrayOf(BodyPart.leg)
    }

    companion object
    {
        val POWER_ID: String = DataManager.MakeTextID(LostBodyPart_Leg::class.java)
    }
}
